articles/quickstart/backend/webapi-owin/01-authorization.md

```diff
@@ -39,16 +35,37 @@ The sample code has an `appsettings` section in `Web.config` which configures it
 
 ### Install dependencies
 
-To use Auth0 Access Tokens with ASP.NET Core you will use the OWIN JWT Middleware which is available in the `Microsoft.Owin.Security.Jwt` NuGet package. Also install the `Auth0.OpenIdConnectSigningKeyResolver` NuGet package which will assist you in verifying the token signature.
+To use Auth0 Access Tokens with ASP.NET you will use the OWIN JWT Middleware which is available in the `Microsoft.Owin.Security.Jwt` NuGet package.
 
 ```bash
 Install-Package Microsoft.Owin.Security.Jwt
-Install-Package Auth0.OpenIdConnectSigningKeyResolver
 ```
 
-### Configuration
+### Verifying the token signature
+As the OWIN JWT middleware doesn't use Open ID Connect Discovery by default, you will need to provide a custom `IssuerSigningKeyResolver`. Such a custom resolver was previously published as part of the `Auth0.OpenIdConnectSigningKeyResolver` package through Nuget. As this package is not available anymore because it was only a thin wrapper, you will need to provide this yourself.
 
-The `Auth0.OpenIdConnectSigningKeyResolver` package which you installed will automatically download the JSON Web Key Set which was used to sign the RS256 tokens by interrogating the OpenID Connect Configuration endpoint (at `/.well-known/openid-configuration`). You can then use it subsequently to resolve the Issuer Signing Key, as will be demonstrated in the JWT registration code below.
+```javascript
+// Support/OpenIdConnectSigningKeyResolver.cs
```

Comment:
If you take the above suggestion with the intro for creating a new file, feel free to remove this line.